webbeans-jetty9/src/it/module/pom.xml

```diff
@@ -0,0 +1,303 @@
+                            <groupId>org.apache.openwebbeans</groupId>
+                            <artifactId>openwebbeans-jetty9</artifactId>
+                            <version>@project.version@</version>
+                        </dependency>
+
+                        <dependency>
+                            <groupId>org.apache.geronimo.specs</groupId>
+                            <artifactId>geronimo-jcdi_2.0_spec</artifactId>
+                            <version>1.0</version>
+                        </dependency>
+
+                        <dependency>
+                            <groupId>org.apache.geronimo.specs</groupId>
+                            <artifactId>geronimo-atinject_1.0_spec</artifactId>
+                            <version>1.0</version>
+                        </dependency>
+
+                        <dependency>
+                            <groupId>org.apache.geronimo.specs</groupId>
+                            <artifactId>geronimo-interceptor_1.2_spec</artifactId>
+                            <version>1.0</version>
+                        </dependency>
+                    </dependencies>
+                </plugin>
+
+            </plugins>
+        </pluginManagement>
+
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.1</version>
+                <executions>
+                    <execution>
+                        <id>jetty-distribution</id>
+                        <phase>generate-test-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.eclipse.jetty</groupId>
+                                    <artifactId>jetty-distribution</artifactId>
+                                    <version>@jettyVersion@</version>
+                                    <type>zip</type>
+                                    <overWrite>false</overWrite>
+                                    <outputDirectory>${project.build.directory}</outputDirectory>
+                                    <excludes>*/demo-base/**</excludes>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>@project.groupId@</groupId>
+                                    <artifactId>@project.artifactId@</artifactId>
+                                    <version>@project.version@</version>
+                                    <type>jar</type>
+                                    <classifier>config</classifier>
+                                    <overWrite>true</overWrite>
+                                    <outputDirectory>${project.build.directory}/jetty-distribution-@jettyVersion@</outputDirectory>
```

Comment:
I really wish dependency:unpack let you strip leading directories so I could force jetty.home to a specific directory